### ISSUE TITLE
CSS custom properties error

### ISSUE THREAD

**Issue by JayPanoz** [w3c/epubcheck#790]
TL;DR: when validating an EPUB file using CSS custom properties, a.k.a. CSS variables ([spec](https://www.w3.org/TR/css-variables/)), you get the following error: 

```
An error occurred while parsing the CSS: Token '-' not allowed here, expecting a property name.
```

It is my understanding this error occurs because epubcheck doesn’t support EPUB 3.1 yet but the spec is part of the [CSS Snapshot 2017](https://www.w3.org/TR/CSS/) (with a “rough interoperability” disclaimer though).

As the main [Readium CSS designer and maintainer](https://github.com/readium/readium-css/tree/master), I can tell a significant number of implementations are likely to ship with [CSS custom properties for user settings](https://gist.github.com/JayPanoz/c5bbf0bd7e53997d3a91d9c1be44a82f). There is also nothing we can do to prevent exposure to authors if needed.

**Comment by rakutenjeff** [comment]
Yes I encountered this as well.

**Comment by rdeltour** [comment]
This is clearly a bug, as it creates false-positives.

**Comment by JayPanoz** [comment]
@rdeltour If needed, you can use ReadiumCSS docs as a test file: 
https://github.com/readium/readium-css/raw/master/docs/ReadiumCSS_docs.epub

The interesting part of this file is that custom properties are in a feature query (`@supports`) so they’re kinda not exposed directly.
